.github/copilot-instructions.md

```diff
@@ -0,0 +1,173 @@
+# GEDCOM Registries Repository
+
+GEDCOM Registries is a Python-based repository containing YAML files that define GEDCOM concepts and structures. The repository validates YAML files against JSON schemas and generates summary files.
+
+Always reference these instructions first and fallback to search or bash commands only when you encounter unexpected information that does not match the info here.
+
+## Working Effectively
+
+### Bootstrap, Build, and Validate the Repository
+
+Follow these steps in exact order for any fresh clone:
+
+1. **Initialize Git Submodules** (REQUIRED FIRST STEP):
+   ```bash
+   git submodule update --init --recursive
+   ```
+   Takes ~5 seconds. NEVER CANCEL. This downloads GEDCOM.io and GEDCOM repositories needed for validation schemas.
+
+2. **Install Python Dependencies** (if not already available):
+   ```bash
+   pip3 install PyYAML jsonschema
+   ```
+   PyYAML and jsonschema are required. yamllint is typically pre-installed.
+
+3. **Run Full Validation Workflow**:
+   ```bash
+   # YAML syntax validation (2 seconds)
+   yamllint .
+   
+   # Schema validation (9 seconds) - NEVER CANCEL
+   cd registry_tools
+   find .. -type f -name "*.yaml" -not -path "../registry_tools/*" -exec dirname {} \; | sort -u | while read -r dir; do
+     yaml_files=("$dir"/*.yaml)
+     if [ "${#yaml_files[@]}" -gt 0 ]; then
+       echo "Checking directory $dir"
+       python3 validator.py "${yaml_files[@]}"
+       if [ $? -ne 0 ]; then
+         exit 1
+       fi
+     fi
+   done
+   cd ..
```

Comment:
```suggestion
```

The bash script to run individual file validation is unnecessary because `global_validator.py` does all the single-file checks (with less Python startup overhead); see line 57 of `global_validator.py` for where this happens